app/src/main/res/values/localpages.xml

```diff
@@ -3,217 +3,12 @@
-    <string name="error_file_access_denied_message"><![CDATA[
-<ul>
-<li>It may have been removed, moved, or file permissions may be preventing access.</li>
-</ul>
-     ]]></string>
-
-    <!-- NavigationDelegate.ERROR_CATEGORY_URI -->
-    <string name="error_proxy_connection_refused_title">Proxy Server Refused Connection</string>
-    <string name="error_proxy_connection_refused_message"><![CDATA[
-<p>The browser is configured to use a proxy server, but the proxy refused a connection.</p>
-<ul>
-<li>Is the browser\'s proxy configuration correct? Check the settings and try again.</li>
-<li>Does the proxy service allow connections from this network?</li>
-<li>Still having trouble? Consult your network administrator or Internet provider for assistance.</li>
-</ul>
-     ]]></string>
-
-    <string name="error_unknown_proxy_host_title">Proxy Server Not Found</string>
-    <string name="error_unknown_proxy_host_message"><![CDATA[
-<p>The browser is configured to use a proxy server, but the proxy could not be found.</p>
-<ul>
-<li>Is the browser\'s proxy configuration correct? Check the settings and try again.</li>
-<li>Is the computer connected to an active network?</li>
-<li>Still having trouble? Consult your network administrator or Internet provider for assistance.</li>
-</ul>
-     ]]></string>
-
-    <!-- NavigationDelegate.ERROR_CATEGORY_SAFEBROWSING -->
-    <string name="error_safe_browsing_malware_uri_title">Malware site issue</string>
-    <string name="error_safe_browsing_malware_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as an attack site and has been blocked based on your security preferences.</p>
-     ]]></string>
-
-    <string name="error_safe_browsing_unwanted_uri_title">Unwanted site issue</string>
-    <string name="error_safe_browsing_unwanted_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as serving unwanted software and has been blocked based on your security preferences.</p>
-     ]]></string>
-
-    <string name="error_safe_harmful_uri_title">Harmful site issue</string>
-    <string name="error_safe_harmful_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as a potentially harmful site and has been blocked based on your security preferences.</p>
-     ]]></string>
-
-    <string name="error_safe_phishing_uri_title">Deceptive site issue</string>
-    <string name="error_safe_phishing_uri_message"><![CDATA[
-<p>This web page at %1$s has been reported as a deceptive site and has been blocked based on your security preferences.</p>
-     ]]></string>
-
     <!-- Private Browsing -->
     <string name="private_browsing_title">Private Browsing</string>
     <string name="private_browsing_body"><![CDATA[
 <h1>Private Browsing</h1>
 
-<p>When you browse in a Private Window, Firefox Reality <strong>does not save</strong>:</p>
+<p>When you browse in a Private Window, %appName% <strong>does not save</strong>:</p>
```

Comment:
I don't think that %appName% is supported  by our l10n tooling (see [this comment](https://github.com/MozillaReality/FirefoxReality/pull/626#discussion_r225403197)). Could you please use %1$S instead?
Otherwise I think this is good to go :) thanks!

Reply:
Fixed.